templates/cloud_config.j2

```diff
@@ -24,6 +24,26 @@ write_files:
     permissions: 0744
     content: |
       {{ lookup('file', role_path + '/files/healthcheck.sh') | indent(6, False) }}
+
+  {% if wimpy_aws_cloudwatch_enabled -%}
+  - path: /home/core/cloudwatch.conf
+    owner: core:core
+    permissions: 0744
+    content: |
+      log_group = "/{{ wimpy_deployment_environment }}/{{ wimpy_project_name }}"
+      state_file = "/var/logs/journald-cloudwatch-logs/state"
+      log_priority = {{ wimpy_aws_logs_priority }}
+  - path: /home/core/install_cloudwatch_agent
+    owner: core:core
+    permissions: 0744
+    content: |
+      #!/bin/sh
+      mkdir -p /opt/bin
+      mkdir -p /var/logs/journald-cloudwatch-logs/state
+      curl -L https://github.com/advantageous/systemd-cloud-watch/releases/download/v0.2.1/systemd-cloud-watch_linux > /opt/bin/journald-cloudwatch-logs
```

Comment:
This installs a tool that sends journald logs (logs coming from systemd) to CloudWatch